python/3d/auto_generated/qgs3dmapsettings.sip.in

```diff
@@ -79,6 +83,29 @@ to improve the precision.
     QgsVector3D origin() const;
 %Docstring
 Returns coordinates in map CRS at which 3D scene has origin (0,0,0)
+%End
+
+    void setOrientation( double newOrientationZ );
+%Docstring
+Sets the z orientation of the 3d scene
+
+:param newOrientationZ: orientation in degrees
```

Comment:
Also note if this is clockwise / counterclockwise

Reply:
Done.